Sources/Plasma/PubUtilLib/plGImage/plPNG.cpp

```diff
@@ -226,27 +229,72 @@ bool plPNG::IWrite(plMipmap* source, hsStream* outStream)
             row_ptrs[i] = (png_bytep)srcp + (i * stride);
         }
 
+        // Write image data
         png_write_image(png_ptr, row_ptrs);
+
+        // Prepare Textual Metadata
+        char time_string[26]; // RFC-1123: "Day, DD-Mon-YYYY hh:mm:ss";
+        time_t rawtime = time(nullptr);
+        strftime(time_string, sizeof(time_string), "%a, %d-%b-%Y %T", gmtime(&rawtime));
```

Comment:
Is this limited to 25 chars in all locales, or only "C" / "en_*"?  If the latter, we need to either force the locale to "C" or expand the string length...

Reply:
There doesn't appear to be a limit to the field content length.  It simply needs to be NULL-terminated.

What would you recommend for determining the temporary string size here?

While looking into this to answer, I think I discovered something about the specs that might save some headaches.  It appears we can use an International Text (iTXT) chunk for any of the fields, and that allows Unicode.  Testing confirms this works as expected.

Reply:
`strftime` is still fine to use (and it looks like we'll have to wait for C++20 to get an STL equivalent with `std::chrono`).  I was just suggesting a larger buffer, or a format (or locale) that's not likely to change based on the user's language setting.

Reply:
That format is suggested (but not strictly required) by the specifications, so it seemed the best choice.